flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/expressions/DecimalTypeTest.scala

```diff
@@ -295,32 +310,1042 @@ class DecimalTypeTest extends ExpressionTestBase {
       "f4 < f0",
       "true")
 
-    // TODO add all tests if FLINK-4070 is fixed
     testSqlApi(
       "12 < f1",
       "true")
   }
 
+  @Test
+  def testTableDataSource(): Unit = {
```

Comment:
`testFieldAcess`? There is no DataSource here.

Reply:
These cases using to test selected column from table source in ITcase, I think ’testFieldAcess' make sense too。